Failed to get the config file

**Describe the bug**
Just updated to 0.5.0 and now I get the following error message during startup:

```
mschop@mschop-work:~/Programs/teams-for-linux$ yarn start
yarn run v1.17.3
$ yarn install && electron ./app
[1/4] Resolving packages...
success Already up-to-date.
$ electron-builder install-app-deps
  • electron-builder  version=21.1.5
  • loaded configuration  file=package.json ("build" field)
  • rebuilding native dependencies  dependencies=spellchecker@3.7.0 platform=linux arch=x64
  • rebuilding native dependency  name=spellchecker version=3.7.0
configPath = /home/mschop/.config/Electron
Failed to get the config file { Error: Cannot find module '/home/mschop/.config/Electron/config.json'
    at Module._resolveFilename (internal/modules/cjs/loader.js:584:15)
    at Function.Module._resolveFilename (/home/mschop/Programs/teams-for-linux/node_modules/electron/dist/resources/electron.asar/common/reset-search-paths.js:43:12)
    at Function.Module._load (internal/modules/cjs/loader.js:510:25)
    at Module.require (internal/modules/cjs/loader.js:640:17)
    at require (internal/modules/cjs/helpers.js:20:18)
    at getConfigFile (/home/mschop/Programs/teams-for-linux/app/config/index.js:89:10)
    at argv (/home/mschop/Programs/teams-for-linux/app/config/index.js:6:19)
    at Object.<anonymous> (/home/mschop/Programs/teams-for-linux/app/index.js:2:35)
    at Object.<anonymous> (/home/mschop/Programs/teams-for-linux/app/index.js:22:3)
    at Module._compile (internal/modules/cjs/loader.js:693:30) code: 'MODULE_NOT_FOUND' }
configFile = {}
[22176:0807/100156.997291:ERROR:CONSOLE(5566)] "SyntaxError: Unexpected token Î in JSON at position 4972212", source: chrome-devtools://devtools/bundled/shell.js (5566)
```

**To Reproduce**
Steps to reproduce the behavior:
1. Execute `yarn start`


**Expected behavior**
The error message should not occur. The config file should be created with useful defaults.

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04
 - Installation package: source
 - Version 0.5.0

--- a/app/config/index.js
+++ b/app/config/index.js
@@ -88,7 +88,7 @@ function getConfigFile(configPath) {
 	try {
 		return require(path.join(configPath, 'config.json'));
 	} catch (e){
-		console.error('Failed to get the config file', e);
+		console.info('Failed to get the config file, using default values');
 		return {};
 	}
 }

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "teams-for-linux",
-  "version": "0.5.1",
+  "version": "0.6.0",
   "main": "app/index.js",
   "description": "Unofficial client for Microsoft Teams for Linux",
   "homepage": "https://github.com/IsmaelMartinez/teams-for-linux",

